lib/src/migrators/module.dart

```diff
@@ -361,32 +367,45 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
       } else {
         // TODO(jathak): Handle the case where the expression of this
         // declaration has already been patched.
-        addPatch(patchDelete(variable.span));
+        addPatch(
+            patchDelete(variable.span, start: -variable.span.start.column));
```

Comment:
It's unlikely, but this could break if the variable is on a line with other code. Maybe teach `patchDelete()` to delete the entire line if all that's left is whitespace?